Makefile

```diff
@@ -21,9 +21,13 @@ tests:
 .PHONY: tests
 
 bench: all
-	@ulimit -s unlimited && perf stat _build/default/benchmark/church.exe  > /dev/null 2>  perf.log
-	@ulimit -s unlimited && perf stat _build/default/benchmark/scott.exe   > /dev/null 2>> perf.log
-	@ulimit -s unlimited && perf stat _build/default/benchmark/lambdas.exe > /dev/null 2>> perf.log
+	@echo "to allow perf, run echo 0 > /proc/sys/kernel/perf_event_paranoid"
```

Comment:
This is too system-specific, I think. I would just revert the changes to this file (also not related to the MR).

Reply:
To me is related to "make sure the example compiles" and it perf is not allowed you get a strange "255" error code. I wanted to make sure the user get a bit of information. I am replacing by something less specific now.